defs/go.bzl

```diff
@@ -104,6 +105,7 @@ go_genrule = rule(
         "message": attr.string(),
         "executable": attr.bool(default = False),
         "_go_context_data": attr.label(default = Label("@io_bazel_rules_go//:go_context_data")),
+        "_go_stdlib": attr.label(default = Label("@io_bazel_rules_go//:stdlib")),
```

Comment:
This is the wrong thing to do, instead you declare
```
go_genrule = go_rule(
```
instead, it adds in all the attributes and toolchains you need to build a go context correctly.
This was done specifically to allow us to make changes without breaking people, it should reduce the churn you suffer (you also need the builders attribute now)

Reply:
ah, ok, I will look into that.